tests/test_ribametamod.py

```diff
@@ -0,0 +1,322 @@
+    metaswap_lookup_table: Path,
+    bucket_ribametamod_loc: Path,
+) -> None:
+    """
+    Test if coupled ribametamod models run with the iMOD Coupler development version.
+    """
+    path_dev = tmp_path_dev / "bucket_model_driver_metamod"
+
+    shutil.copytree(bucket_ribametamod_loc, path_dev)
+
+    toml_path = path_dev / "imod_coupler.toml"
+    modflow6_dll = modflow_dll_devel
+    modflow6_model_dir = path_dev / "modflow6"
+    metaswap_dll = metaswap_dll_devel
+    metaswap_dll_dependency = metaswap_dll_dep_dir_devel
+    metaswap_model_dir = path_dev / "metaswap"
+
+    fill_para_sim_template(path_dev / "metaswap", metaswap_lookup_table)
+
+    write_metamod_toml(
+        path_dev,
+        modflow6_dll,
+        modflow6_model_dir,
+        metaswap_dll,
+        metaswap_dll_dependency,
+        metaswap_model_dir,
+    )
+    run_coupler(toml_path)
+
+
+def test_ribamod(
+    tmp_path_dev: Path,
+    modflow_dll_devel: Path,
+    ribasim_dll_devel: Path,
+    ribasim_dll_dep_dir_devel: Path,
+    bucket_ribametamod_loc: Path,
+) -> None:
+    """
+    Test if coupled ribametamod models run with the iMOD Coupler development version.
+    """
+    path_dev = tmp_path_dev / "bucket_model_driver_ribamod"
+
+    shutil.copytree(bucket_ribametamod_loc, path_dev)
+
+    toml_path = path_dev / "imod_coupler.toml"
+    modflow6_dll = modflow_dll_devel
+    modflow6_model_dir = path_dev / "modflow6"
+
+    write_ribamod_toml(
+        path_dev,
+        modflow6_dll,
+        modflow6_model_dir,
+        ribasim_dll_devel,
+        ribasim_dll_dep_dir_devel,
+        path_dev / "ribasim" / "ribasim.toml",
+    )
+    run_coupler(toml_path)
+
+
+def test_ribametamod(
+    tmp_path_dev: Path,
+    modflow_dll_devel: Path,
+    metaswap_dll_devel: Path,
+    metaswap_dll_dep_dir_devel: Path,
+    metaswap_lookup_table: Path,
+    ribasim_dll_devel: Path,
+    ribasim_dll_dep_dir_devel: Path,
+    bucket_ribametamod_loc: Path,
+) -> None:
+    """
+    Test if coupled ribametamod models run with the iMOD Coupler development version.
+    """
+    path_dev = tmp_path_dev / "bucket_model_driver_ribametamod"
+
+    shutil.copytree(bucket_ribametamod_loc, path_dev)
+
+    fill_para_sim_template(path_dev / "metaswap", metaswap_lookup_table)
+
+    # toml_path = path_dev / "imod_coupler.toml"
+    modflow6_dll = modflow_dll_devel
+    modflow6_model_dir = path_dev / "modflow6"
+    metaswap_dll = metaswap_dll_devel
+    metaswap_dll_dependency = metaswap_dll_dep_dir_devel
+    metaswap_model_dir = path_dev / "metaswap"
+
+    write_ribametamod_toml(
+        path_dev,
+        modflow6_dll,
+        modflow6_model_dir,
+        metaswap_dll,
+        metaswap_dll_dependency,
+        metaswap_model_dir,
+        ribasim_dll_devel,
+        ribasim_dll_dep_dir_devel,
+        path_dev / "ribasim" / "ribasim.toml",
+    )
+    # run_coupler(toml_path) # don't run until optional branch is merged
+
+
+def write_metamod_toml(
```

Comment:
Can't you use primod for this?

Reply:
At this moment not, since we don't have an instance of Ribamod or Metamod. Eventually we want to use an instance of ribametamod for this indeed.

Reply:
Okay :)

Maybe add a TODO comment in front of the functions that this should use primod in the long run?